### PR TITLE
Fix .desktop file

### PR DESCRIPTION
**Pull request by nE0sIghT**
Trivial change to match uppercase conversion

### CODE REVIEW

File: linux_various/PCSX2.desktop.in [hunk 1]
```diff
@@ -5,7 +5,7 @@ Type=Application
 Name=PCSX2
 GenericName=Playstation 2 Emulator
 Comment=Sony Playstation 2 emulator
-Exec=pcsx2
-Icon=pcsx2
+Exec=PCSX2
```
Comment (gregory38) [inline linux_various/PCSX2.desktop.in]: I want people to execute PCSX2-linux.sh, this way nvidia variable is correctly set (if you have a better solution I take it)

**Comment by nE0sIghT** [comment]
No, i do not have a better solution. I want to avoid LD_LIBRARY_PATH magic in .sh wrapper for Gentoo packaging (because it can break applications in weird ways - https://bugs.gentoo.org/show_bug.cgi?id=531990 for example) as well as "/usr/bin/which" dependency.

Maybe create separate tiny wrapper for packaging? Something like
```sh
MY_LD_LIBRARY_PATH="${LD_LIBRARY_PATH}"

# openSUSE don't follow FHS !!!!
if [ -e /etc/SuSE-release ]; then
    MY_LD_LIBRARY_PATH="${MY_LD_LIBRARY_PATH:+$MY_LD_LIBRARY_PATH:}"/usr/lib/wx-2.8-stl
fi

# And finally launch me
LD_LIBRARY_PATH="${MY_LD_LIBRARY_PATH}" __GL_THREADED_OPTIMIZATIONS=1 /usr/games/PCSX2 $@
```

**Comment by gregory38** [comment]
Well the probability to have /usr/lib/wx-2.8-stl on gentoo is null. And if you have it you likely need it. However the 3rdparty can be commented for the moment.

which won't be called normally. Maybe you can add a -x.

Note (just to be sure): LD_LIBRARY_PATH is only changed for PCSX2. It won't change behavior of others application.

**Comment by nE0sIghT** [comment]
Btw, maybe set __GL_THREADED_OPTIMIZATIONS directly in .desktop file?

**Comment by gregory38** [comment]
Well if is possible, why not?

**Comment by nE0sIghT** [comment]
Pushed new commits

**Comment by gregory38** [comment]
Thanks very much.

**Pull request merged**